sources/packages/web/src/services/http/dto/ProgramInfoRequest.dto.ts

```diff
@@ -43,13 +43,38 @@ export interface PIRDeniedReasonAPIOutDTO {
   description: string;
 }
 
+export interface PIRSearchCriteria {
+  search?: string;
+  intensityFilter?: string[];
+  page: number;
+  pageLimit: number;
+  sortField?: string;
+  sortOrder?: "ASC" | "DESC";
+}
+
 export interface PIRSummaryAPIOutDTO {
   applicationNumber: string;
   studyStartPeriod: string;
   studyEndPeriod: string;
   applicationId: number;
-  pirStatus: string;
+  pirStatus: ProgramInfoStatus;
   fullName: string;
+  submittedDate: string;
+  givenNames: string;
+  lastName: string;
+  studentNumber: string;
+  studyIntensity: OfferingIntensity;
+  program: string;
+  applicationData?: {
+    programName?: string;
+    startDate?: string;
+    endDate?: string;
+  };
+  offeringData?: {
+    programName?: string;
+    startDate?: string;
+    endDate?: string;
+  };
```

Comment:
As mentioned in the API comment, these value can come from API as final values which can be used at the UI.